app/model/editions/EditionsFront.scala

```diff
@@ -1,14 +1,22 @@
 package model.editions
 
+import org.postgresql.util.PGobject
 import play.api.libs.json.Json
 import scalikejdbc.WrappedResultSet
 
-case class EditionsFrontMetadata(nameOverride: Option[String])
-
 object EditionsFrontMetadata {
   implicit val format = Json.format[EditionsFrontMetadata]
 }
 
+case class EditionsFrontMetadata(nameOverride: Option[String], swatch: Option[Swatch]) {
+  def toPGobject: PGobject = {
+    val pgo = new PGobject()
+    pgo.setType("json")
```

Comment:
Should this be JSONB? If it's sent as JSON won't the DB have to cast it?

Reply:
Yes, fixed.  Also fixed in skeleton.